junit-platform-console/src/main/java/org/junit/platform/console/options/CommandLineOptions.java

```diff
@@ -31,9 +31,11 @@
 @API(Internal)
 public class CommandLineOptions {
 
+	static final Details DEFAULT_DETAILS = Details.TREE;
```

Comment:
Will change this to `--hide-details` behaviour, namely: `static final Details DEFAULT_DETAILS = Details.NONE;`